ci/provision-cluster.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+
+if [[ -d /opt/cluster ]]; then
+  echo "✔ Cluster already installed"
+  exit 0
+fi
+
+(
+  set -x
+  cd /opt
+  git clone https://github.com/liquidinvestigations/cluster
+  cd /opt/cluster
+  cp examples/cluster.ini ./
+  sed -i '/nomad_meta/a vmck_worker = true' cluster.ini
+  sed -i '/^run_jobs =.*/c\run_jobs = dnsmasq' cluster.ini
+  chown -R vagrant: .
```

Comment:
Q: We need this?

Reply:
VMs only run on nomad nodes that have the `vmck_worker` tag, so if we want to test the vmck workflow we need it. As for the `dnsmasq` I haven't tested it but we could do without.